### PR TITLE
Add multipart-form data test case with expect header

### PR DESCRIPTION
**Pull request by chamil321**

### CODE REVIEW

File: tests/ballerina-integration-test/src/test/resources/http/httpservices/100_continue.bal [hunk 1]
```diff
@@ -38,4 +39,22 @@ service<http:Service> helloContinue bind { port: 9090 } {
...
+        methods: ["POST"]
+    }
+    getFormParam(endpoint caller, http:Request req) {
+        string replyMsg = "Result =";
+        mime:Entity[] bodyParts = check req.getBodyParts();
+        int i = 0;
+        while (i < lengthof bodyParts) {
+            mime:Entity part = bodyParts[i];
+            mime:ContentDisposition contentDisposition = part.getContentDisposition();
+            replyMsg += " Key:" + contentDisposition.name + " Value: " + check part.getBodyAsString();
```
Comment (rasika) [inline tests/ballerina-integration-test/src/test/resources/http/httpservices/100_continue.bal]: Better if we could use a `StringBuilder` inside the while loop.

Comment (chamil321) [reply tests/ballerina-integration-test/src/test/resources/http/httpservices/100_continue.bal]: Does ballerina have a `StringBuilder`? :open_mouth:

Comment (Shan1024) [reply tests/ballerina-integration-test/src/test/resources/http/httpservices/100_continue.bal]: AFAIK, we don't have.

Comment (chamil321) [reply tests/ballerina-integration-test/src/test/resources/http/httpservices/100_continue.bal]: Yeah, I also not aware of come feature.

Comment (rasika) [reply tests/ballerina-integration-test/src/test/resources/http/httpservices/100_continue.bal]: Ops, didn't see that this is a .bal file. Let's stick into this. Anyway it is nice to have such a feature ;)

**Pull request merged**